Seminar2/task3.py

```diff
@@ -0,0 +1,29 @@
+num1 = str(input('Введите первую десятичную дробь вида a/b: '))
+num2 = str(input('Введите вторую десятичную дробь вида a/b: '))
```

Comment:
`input` уже возвращает строку
```suggestion
num1 = input('Введите первую десятичную дробь вида a/b: ')
num2 = input('Введите вторую десятичную дробь вида a/b: ')
```